packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -137,6 +135,9 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         frameworkController.onConnectVmEvent.listen(_connectVm);
     _showPageSubscription =
         frameworkController.onShowPageId.listen(_showPageById);
+
+    cancel();
```

Comment:
cancel in initState seems very strange. why do we need this?

Reply:
Ah. copied this pattern from what we do in other places where we call `addAutoDisposeListener`, but now I realize those are calls in didChangeDependencies where the listener could be added twice. removed.